### ISSUE TITLE
`error: 'mac/hid.c' file not found` on m1 macOS

### ISSUE THREAD

**Issue by chenrui333** [trezor/trezord-go#286]
Got some build error when trying to build [trezor-bridge 2.0.32](https://github.com/Homebrew/homebrew-core/pull/113890)

```
usb/lowlevel/hidapi/hid.go:25:11: fatal error: 'mac/hid.c' file not found
        #include "mac/hid.c"
                       ^~~~~~~~~~~
1 error generated.
```

What would be the way to solve the issue? Thanks!

**Comment by prusnak** [comment]
You are not using `git clone --recursive` to fetch submodules.

Readme fixed in 8f35971d3c36ea8b91ff54810397526ef8e741c5

**Comment by chenrui333** [comment]
Oh, I see, thanks!
